### ISSUE TITLE
Contant variable issue.

### ISSUE THREAD

**Issue by honest888** [ZFC-Digital/puppeteer-real-browser#251]
https://github.com/ZFC-Digital/puppeteer-real-browser/blob/abe5bb17f8b1660c8314d43bf09015d432e24378/lib/cjs/index.js#L79

Your new version has an issue in this line because puppeteer is contant variable. 
`const puppeteer = require("rebrowser-puppeteer-core"); //This is contant variable.
...
if (plugins.length > 0) {
    const { addExtra } = await import("puppeteer-extra");

    puppeteer = addExtra(puppeteer);

    for (const item of plugins) {
      puppeteer.use(item);
    }
  }
`
I think, you have to update this.

**Comment by firdyfirdy** [comment]
That's right! I've been looking around for the error but after I saw the latest commit with id: c7920f3, it's changes the first line from var to const.
